### ISSUE TITLE
Am I able to return the mask for the Face as well as Hair

### ISSUE THREAD

**Issue by SpeedOfSpin** [YBIGTA/pytorch-hair-segmentation#39]
Thank you for such great work.

Is it possible that I run this model to get segmentation for BOTH face and hair as in the attached picture?

Many thanks
sos
![48294893-bc640b80-e4ca-11e8-81d0-7f64af8b3c16](https://user-images.githubusercontent.com/6115268/56302806-6870e900-6132-11e9-8d08-7efab040ba40.png)

**Comment by austingg** [comment]
@SpeedOfSpin  you should retrain the model with face, hair and other. Currently it only segment hair

**Comment by jacquelinelala** [comment]
> Thank you for such great work.
> 
> Is it possible that I run this model to get segmentation for BOTH face and hair as in the attached picture?
> 
> Many thanks
> sos
> ![48294893-bc640b80-e4ca-11e8-81d0-7f64af8b3c16](https://user-images.githubusercontent.com/6115268/56302806-6870e900-6132-11e9-8d08-7efab040ba40.png)

Hi, could you tell the arxiv link or paper title about your question? 
Thanks a lot.

**Comment by madhurgoel** [comment]
Pleas have a look at https://github.com/tensorflow/models/tree/master/research/deeplab.
Could you please tell me what dataset you are using for both hair and face masks ?
